InfiniteRecharge/src/main/java/frc/robot/Robot.java

```diff
@@ -54,23 +58,22 @@ public void robotInit() {
         config = ConfigChooser.getConfig();
 
         visionSubsystem = new VisionSubsystem(config);
-        visionSubsystem.initialize();
-
         navigationSubsystem = new NavigationSubsystem(config, visionSubsystem);
-        navigationSubsystem.initialize();
-
         driveSubsystem = new DriveSubsystem(config, navigationSubsystem, oi);
-        driveSubsystem.initialize();
-        navigationSubsystem.setDrive(driveSubsystem); // Java

```

Comment:
@nitish123-bitbuckets make sure you still call this `navigationSubsystem.setDrive(driveSubsystem);` We still need it. Don't put it in the loop, just put it after the constructor.